packages/@aws-cdk/aws-config/lib/rule.ts

```diff
@@ -274,6 +279,15 @@ export class ManagedRule extends RuleNew {
 
     this.isManaged = true;
   }
+
+  private renderScope(): CfnConfigRule.ScopeProperty | undefined {
```

Comment:
I think you misunderstood my request before, or missed my comment. Why not extract out into an unexported function, rather than copy/paste across the two usages?

Reply:
i did misunderstand, moved it into a function local to this file